### DISCUSSION TITLE
Forma correcta de cambiar el tamaño de una imagen ¿?

### DISCUSSION THREAD

**Discussion by axlgoze** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#322] [Q&A] [answered]
Hola alguien puede ayudarme a saber como se modifica el tamaño de una imagen... me cuesta trabajo entender si se hace con atributos width y height en la etiqueta img o se hace en el archivo de css por medio de una clase ... ¿es lo mismo, cual es mejor? y otra duda que tengo es cual es la diferencia de poner width en porcentaje (width: 100%) y en pixeles (width:100px) ??? 🤔 🆘

**Comment by OmarVenturaP** [comment, marked as answer]
Hola, de ambas formas es correcta, solo que para tener más organización de tu código se sugiere hacerlo en un archivo CSS. 

La forma de hacerlo depende de lo que quieres lograr.
La diferencia de % y PX es el tamaño de la imagen. Ya que al ponerle porcentaje agarra el tamaño del padre.

Ej. Supongamos que el elemento padre tiene un width de 200 px

Si a la imagen le pones width 100%, entonces el tamaño va a ser el del padre, es decir 200px.

Pero si le pones 100px, entonces el tamaño será de 100px y en porcentaje sería 50%.

No sé si me pude explicar 😅

**Reply by axlgoze** [reply]
Si !! perfecto! Gracias ✌

**Comment by martinGM05** [comment]
Eso es más que nada cuestiones de buenas prácticas, lo recomendable es desde css ya que puede que otra imagen necesite esos mismos atributos y evitar código duplicado. Sin embargo si es el caso que no no importa si está dentro de la etiqueta los estilos. y con respecto a tu otra pregunta pues depende del tamaño de donde se encuentre la imagen.  El 100% estás diciendo que ocuparás todo el espacio disponible y 100px es que solo ocupará 100 pixeles.

**Comment by doguedogue** [comment]
Estas en lo correcto, si lo haces desde la imagen estas modificando sus atributos.

Si lo haces desde el CSS estas modificando el estilo de todos los componente (selector, clase o id)

el porcentaje es respecto al tamaño de la pantalla donde se visualiza tu imagen.
